joomla/joomla-cms
# [4.0] Apply the proposed changes from @brianteeman to the testing plugin language file

Pull Request for Issue https://github.com/joomla/joomla-cms/pull/30555#issuecomment-686334912

### Summary of Changes

Apply the reverted changes from this PR: https://github.com/joomla/joomla-cms/pull/30555

### Testing Instructions

install 4.0-dev
apply this PR
install testing sable data (only aviable in dev builds)
notice the changed language strings

### Actual result BEFORE applying this Pull Request

The text includes `just` that should be avoided

### Expected result AFTER applying this Pull Request

The proposed strings should be improved

### Documentation Changes Required

None.
